src/sh/python-lint/poetry-check.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+POETRY_SUBDIRECTORY=./src/python/hooks
+
+poetry --version >/dev/null 2>&1 || {
+	echo "Poetry is not installed. Please install poetry to continue."
+	exit 1
+}
+
+cd $POETRY_SUBDIRECTORY || exit 1
```

Comment:
Run poetry install here to enforce that the poetry dependencies are installed in the current environment